source/assets/code/api-update-cluster-image-response-json.txt

```diff
@@ -0,0 +1,453 @@
+
+		             },
+		             "f:serverHost":{
+
+		             }
+		          },
+		          "f:secrets":{
+		             ".":{
+
+		             },
+		             "f:users":{
+
+		             }
+		          },
+		          "f:updateStrategy":{
+
+		          }
+		       }
+		    },
+		    "manager":"kubectl",
+		    "operation":"Update",
+		    "time":"2020-07-24T14:27:58Z"
+		 },
+		 {
+		    "apiVersion":"psmdb.percona.com/v1",
+		    "fieldsType":"FieldsV1",
+		    "fieldsV1":{
+		       "f:spec":{
+		          "f:backup":{
+		             "f:containerSecurityContext":{
+		                ".":{
+
+		                },
+		                "f:runAsNonRoot":{
+
+		                },
+		                "f:runAsUser":{
+
+		                }
+		             },
+		             "f:podSecurityContext":{
+		                ".":{
+
+		                },
+		                "f:fsGroup":{
+
+		                }
+		             }
+		          },
+		          "f:clusterServiceDNSSuffix":{
+
+		          },
+		          "f:runUid":{
+
+		          },
+		          "f:secrets":{
+		             "f:ssl":{
+
+		             },
+		             "f:sslInternal":{
+
+		             }
+		          }
+		       },
+		       "f:status":{
+		          ".":{
+
+		          },
+		          "f:conditions":{
+
+		          },
+		          "f:observedGeneration":{
+
+		          },
+		          "f:replsets":{
+		             ".":{
+
+		             },
+		             "f:rs0":{
+		                ".":{
+
+		                },
+		                "f:ready":{
+
+		                },
+		                "f:size":{
+
+		                },
+		                "f:status":{
+
+		                }
+		             }
+		          },
+		          "f:state":{
+
+		          }
+		       }
+		    },
+		    "manager":"percona-server-mongodb-operator",
+		    "operation":"Update",
+		    "time":"2020-07-24T15:35:14Z"
+		 },
+		 {
+		    "apiVersion":"psmdb.percona.com/v1",
+		    "fieldsType":"FieldsV1",
+		    "fieldsV1":{
+		       "f:spec":{
+		          "f:image ":{
+
+		          },
+		          "f:replsets":{
+		             ".":{
+
+		             },
+		             "f:size":{
+
+		             }
+		          }
+		       }
+		    },
+		    "manager":"kubectl",
+		    "operation":"Update",
+		    "time":"2020-07-27T12:21:39Z"
+		 }
+	      ],
+	      "name":"my-cluster-name",
+	      "namespace":"default",
+	      "resourceVersion":"1279853",
+	      "selfLink":"/apis/psmdb.percona.com/v1/namespaces/default/perconaservermongodbs/my-cluster-name",
+	      "uid":"5207e71a-c83f-4707-b892-63aa93fb615c"
+	   },
+	   "spec":{
+	      "allowUnsafeConfigurations":false,
+	      "backup":{
+		 "enabled":true,
+		 "image":"percona/percona-server-mongodb-operator:1.5.0-backup",
+		 "restartOnFailure":true,
+		 "serviceAccountName":"percona-server-mongodb-operator",
+		 "storages":null,
+		 "tasks":null
+	      },
+	      "image":"percona/percona-server-mongodb-operator:1.5.0-mongod4.2",
```

Comment:
why two times? is it correct?

Reply:
it's the only one such place, sure it's a typo... fixed for now